### ISSUE TITLE
Strong reference cycle - objects not released from memory

### ISSUE THREAD

**Issue by alekkova** [aws-amplify/amplify-ui-swift-liveness#129]
### Describe the bug

Strong reference cycle between `CameraPreviewViewModel` and `LivenessCaptureSession` prevents objects to be released from memory after each liveness session. Please see screenshots from Xcode Memory Graph.

![Screenshot 2024-04-18 at 15 27 06](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/155649315/e500b175-d131-481a-ba0e-2600aad3f9f6)

![Screenshot 2024-04-18 at 15 27 18](https://github.com/aws-amplify/amplify-ui-swift-liveness/assets/155649315/558eb0d4-63bc-4097-aa09-58064d681f8f)

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Perform full liveness session flow.
2. Perform another full liveness session flow.
3. Observe Xcode Memory Graph -> objects keep accumulating.
```

### Expected behavior

Objects are released from memory after liveness session is completed.

### Swift Liveness Version

1.2.9

### Xcode version

15.2

### Is this a regression?

No

### OS Version

iOS 17

### Device

iPhone 15

**Comment by phantumcode** [comment]
@alekkova Thanks for submitting the issue.  We'll investigate and provide updates here.

**Comment by phantumcode** [comment]
Fixed in version [1.2.12](https://github.com/aws-amplify/amplify-ui-swift-liveness/releases/tag/1.2.12)
